tests/run/cpp_stl_list.pyx

```diff
@@ -126,14 +106,6 @@ cdef list to_pylist(cpp_list[int]& l):
         incr(it)
     return L
 
-cdef list const_to_pylist(cpp_list[int]& l):
```

Comment:
This looks like it was mistakenly moved. It's used in the old test file and not in the new one.

Reply:
Yes apparently I forgot to move `const_item_ptr_test` as well.